rclip/utils.py

```diff
@@ -48,6 +48,9 @@ def top_arg_type(arg: str) -> int:
 def init_arg_parser() -> argparse.ArgumentParser:
   parser = argparse.ArgumentParser()
   parser.add_argument('query')
+  parser.add_argument('--add', '-a', action='append', default=[], help='phrases to add to the score')
+  parser.add_argument('--subtract', '--sub', '-s', action='append', default=[],
+                      help='phrases to subtract from the score')
```

Comment:
We are not adding phrases to the score, we are modifying "original" query features. It still sounds weird, though :thinking:

```suggestion
  parser.add_argument('--add', '-a', action='append', default=[], help='queries to add to the "original" query')
  parser.add_argument('--subtract', '--sub', '-s', action='append', default=[],
                      help='queries to subtract from the "original" query')
```